test/e2e/delete-doesnt-merge.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { expect, Page } from '@playwright/test';
+import { test } from './electron-test';
+import { createLocalAthensDB, indentLastBlock, saveLastBlock, saveLastBlockAndEnter, unindentLastBlock } from "./utils";
+
+const testSetup = async (page:Page) => {
+    // NOTE this ain't working, and that's ok
+    // await createLocalAthensDB(page, "amazing");
+
+    // await page.pause();
+    // Navigate to daily pages, Click button:nth-child(6)
+    await Promise.all ([
+        page.click('button:nth-child(6)'),
+        page.waitForNavigation()
+    ]);
+
+    // Click textarea
+    await page.click('textarea');
+
+    // Invoke Athena, Press k with modifiers
+    await page.press('textarea', 'Meta+k');
```

Comment:
What I see while debugging is that on linux it hangs waiting for Athena to show up:
```
  pw:api => page.press started +1ms
  pw:api waiting for selector "textarea" +0ms
  pw:api   selector resolved to 2 elements. Proceeding with the first one. +3ms
  pw:api   selector resolved to visible <textarea id="editable-uid-6cbc4d7a0" class="textarea is-e…></textarea> +1ms
  pw:api elementHandle.press("Meta+k") +1ms
  pw:api <= page.press succeeded +14ms
  pw:api => page.fill started +0ms
  pw:api waiting for selector "[placeholder="Find or Create Page"]" +3ms
  pw:api   "networkidle" event fired +88ms
  pw:api => electronApplication.close started +26s
  1) [chromium] › delete-doesnt-merge.spec.ts:50:1 › delete-merge-looses-children ==================

    Timeout of 30000ms exceeded.
```

Maybe the `Meta+k` shortcut doesn't work on linux?

Reply:
`Meta+k` was it